arangod/Agency/Node.h

```diff
@@ -75,6 +75,60 @@ typedef std::chrono::steady_clock::time_point SteadyTimePoint;
 
 class Store;
 
+class SmallBuffer {
+  uint8_t* _start;
+  size_t _size;
+ public:
+  SmallBuffer() : _start(nullptr), _size(0) {}
```

Comment:
this should be marked `noexcept`.
```suggestion
  SmallBuffer() noexcept : _start(nullptr), _size(0) {}
```